Fix and update the calendar app (#6)
SAP-samples/cloud-sdk-team-calendar: exercises/01-getting-started.md

git clone https://github.com/<your-github-username>/cloud-sdk-team-calendar.git
```

## Deploy the S/4HANA and SFSF mock server (Optional)

>Note: If you have access to both SAP S/4HANA and SAP Successfactors Cloud systems, you can skip this step.

Follow these [instructions](https://github.com/SAP/cloud-s4-sdk-book/tree/mock-server#how-to-run-the-server) on how to setup your S4/HANA / SFSF mock server and deploy it on SAP Cloud Platform, Cloud Foundry.

> ## Setup the destination file
> 
> Retrieve the server's route, you can then replace the S4HANA and SFSF `url` in the [`.env`](../.env) file with the route url of the server. The `.env` file should have the same URLs if you use the mock server, because the mock server offers both services:

Reviewer: ```suggestion
## Setup the destinations environment variable

Replace the URL placeholders for the `S4HANA` and `SFSF` destinations in the [`.env`](../.env) file with the URL(s) of your mock server or your SAP S/4HANA and SAP SuccessFactors systems. If you are using the mock server, the URLs for both destinations are the same, because the mock server acts as both, a mocked SAP S4/HANA system and a mocked SAP SuccessFactors system.
```

> ```json
> destinations=[{"name":"S4HANA","url":"<S4-server-url>","authentication":"NoAuthentication"},{"name": "SFSF", "url": "<SFSF-server-url>","authentication":"NoAuthentication"}]
> ```

Reviewer: ```suggestion
```

> > ### If you lost the mock server's route
> > Open the SAP [Cloud Platform Cockpit](https://account.hana.ondemand.com/) and open your subaccount, if you have a trial account it should be called trial, then open your space, where your applications are deployed, you should find the newly deployed mock server.

Reviewer: ```suggestion
> ### If you lost the mock server's route
> Open the SAP [Cloud Platform Cockpit](https://account.hana.ondemand.com/) and open your subaccount, if you have a trial account it should be called trial, then open your space, where your applications are deployed, you should find the newly deployed mock server.
```
```suggestion
> ### If you don't know your mock server's URL
> Open the SAP [Cloud Platform Cockpit](https://account.hana.ondemand.com) and open the subaccount to which the mockserver was deployed. It should be called `trial`, if you are using your trial account. Open the space, where the mock server was deployed. You should find the URL to the mock server here.
```
